frontend/src/components/GoalForm/ReadOnlyObjective.scss

```diff
@@ -0,0 +1,11 @@
+@use '../../colors.scss' as *;
```

Comment:
@AdamAdHocTeam FYI, we don't need to import this unless we are using the colors in the file